srcCxx/ShapeType_main.cxx

```diff
@@ -292,30 +354,26 @@ int run(DomainId_t domain_id, bool use_security,
     int sent_count = 0;
     int recv_count = 0;
     while  ( exit_application == false ) {
-        printf("\nLoop: wait count = %d, sent count = %d, "
-	       "received count = %d\n",
-	       wait_count++, sent_count, recv_count);
+        char msg[256];
+        snprintf(msg, 256, "\nLoop: wait count = %d, sent count = %d, "
```

Comment:
You could check the return value of snprintf

Reply:
updated with return value check